heft-plugins/heft-jest-plugin/includes/jest-preset.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Intentionally included? This doesn't look like it's referenced anywhere.

Reply:
Oops, was testing with this to make sure the changes worked. Will remove